toolium/driver_wrapper.py

```diff
@@ -213,6 +214,20 @@ def connect(self, maximize=True):
                                                                                                 'appium_app_strings'):
             self.app_strings = self.driver.app_strings()
 
+        # Bounds and screen
+        bounds_x = int(self.config.get_optional('Driver', 'bounds_x') or 0)
+        bounds_y = int(self.config.get_optional('Driver', 'bounds_y') or 0)
+
+        monitor_index = int(self.config.get_optional('Driver', 'monitor') or -1)
+        if monitor_index > -1:
+            monitor = screeninfo.get_monitors()[monitor_index]
+            bounds_x += monitor.x
+            bounds_y += monitor.y
+
+        self.driver.set_window_position(bounds_x, bounds_y)
```

Comment:
`set_window_position()` is not implemented in all drivers, at least in Appium. I think that `self.is_maximizable()`, used below, should be checked in this case.

Reply:
Done